src/vs/editor/contrib/suggest/browser/suggestWidgetRenderer.ts

```diff
@@ -167,6 +169,11 @@ export class ItemRenderer implements IListRenderer<CompletionItem, ISuggestionTe
 		data.root.id = getAriaId(index);
 		data.colorspan.style.backgroundColor = '';
 
+		// Set the description of the row to the documentation snippet. This provides a clearer
+		// description as the documentation is announced after list item semantics.
+		data.detailsLabel.id = `${data.root.id}-description`;
```

Comment:
Do we know that `root.id` is always defined?

Reply:
see line 169 above where data.root.id is set within this scope. getAriaId is only dependent on the index.

Reply:
Thanks! Didn't see that.